src/sentry/api/serializers/sessions.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, List
+
+from typing_extensions import TypedDict
+
+DateString = str
+
+
+class QuerySessionGroup(TypedDict):
```

Comment:
Let's make sure these are also used in the implementation of the API. So in future if something changes in the response it reflect documentation too. For example is this the return type from `release_health.backend.run_sessions_query` same as `QuerySessionsResponse`? If so let's add a return type.

Reply:
Great idea! I reused `SessionsQueryResult` type and removed `QuerySessionsResponse` in [b9a8fc1](https://github.com/getsentry/sentry/pull/64032/commits/b9a8fc15bfb1fdec0051ad091c741357522efbf4)